frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,173 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import { PodKind } from '@console/internal/module/k8s';
+import { getName } from '@console/shared/src/selectors/common';
+import {
+  confirmButton,
+  clickKebabAction,
+  goToInstalledOperators,
+  ocsOp,
+  storageClusterRow,
+  storageClusterView,
+  verifyFields,
+} from '../views/add-capacity.view';
+import { CLUSTER_STATUS, EXPAND_WAIT, KIND, NS, POD_NAME_PATTERNS, SECOND } from '../utils/consts';
+import {
+  getOSDPreparePodsCnt,
+  getPodName,
+  getPodPhase,
+  getPodRestartCount,
+  isPodPresent,
+  testPodIsRunning,
+  testPodIsSucceeded,
+} from '../utils/helpers';
+
+const storageCluster = JSON.parse(execSync(`kubectl get -o json -n ${NS} ${KIND}`).toString());
+const cephValue = JSON.parse(execSync(`kubectl get cephCluster -n ${NS} -o json`).toString());
+const clusterStatus = storageCluster.items[0];
+const cephHealth = cephValue.items[0];
+
+const expansionObjects = {
+  clusterJSON: {},
+  previousCnt: 0,
+  updatedCnt: 0,
+  updatedClusterJSON: {},
+  previousPods: { items: [] as PodKind[] },
+  updatedPods: { items: [] as PodKind[] },
+};
+
+describe('Check availability of ocs cluster', () => {
+  if (clusterStatus) {
+    it('Should check if the ocs cluster is Ready for expansion', () => {
+      expect(_.get(clusterStatus, 'status.phase')).toBe(CLUSTER_STATUS.READY);
+    });
+  } else {
+    it('Should state that ocs cluster is not ready for expansion', () => {
+      expect(clusterStatus).toBeUndefined();
+    });
```

Comment:
Why is this check required? If not ready then you either wait for it to get ready or fail the test.

Reply:
If cluster is not ready, instead of making the tests fail, we should skip the test with the message. This is as per the requirement.

Reply:
Are you saying that if the storageCluster is not present then we should skip that the whole of expansion test and progress? You are completely removing the checks in case of no availability. IMO it should fail. For now will let it pass, but please verify.